src/framework/theme/components/select/option-group.component.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { Component, ViewChild } from '@angular/core';
+import { ComponentFixture, fakeAsync, flush, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { RouterTestingModule } from '@angular/router/testing';
+import {
+  NbLayoutModule,
+  NbOptionComponent,
+  NbOptionGroupComponent,
+  NbSelectModule,
+  NbSelectComponent,
+  NbThemeModule,
+} from '@nebular/theme';
+
+@Component({
+  template: `
+    <nb-layout>
+      <nb-layout-column>
+
+        <nb-select [disabled]="selectDisabled">
+          <nb-option-group [disabled]="optionGroupDisabled" [title]="optionGroupTitle">
+            <nb-option *ngIf="showOption" [value]="1" [disabled]="optionDisabled">1</nb-option>
+          </nb-option-group>
+        </nb-select>
+
+      </nb-layout-column>
+    </nb-layout>
+  `,
+})
+export class NbOptionGroupTestComponent {
+  selectDisabled: boolean = false;
```

Comment:
Let's not specify the property type when we assign value during the declaration. I've described that slightly deeper in some comment below.